src/main/java/frc/robot/subsystems/manipulator/ManipulatorIOSparkMax.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems.manipulator;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.RobotController;
+import frc.robot.subsystems.flywheel.FlywheelIO;
+
+public class ManipulatorIOSparkMax implements ManipulatorIO {
+    public static final double GEAR_RATIO = 1;
+    public static final int deviceId = 6; //TODO fill this in
+    private final CANSparkMax motor;
+    private final RelativeEncoder encoder;
+    private final SparkMaxPIDController pid;
+
+    public ManipulatorIOSparkMax(){
+        motor = new CANSparkMax(deviceId, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+        encoder = motor.getEncoder();
+        pid = motor.getPIDController();
+
+        motor.restoreFactoryDefaults();
+
+        motor.setInverted(false);
+
+        motor.enableVoltageCompensation(12.0);
+        motor.setSmartCurrentLimit(30);
+
+        motor.burnFlash();
```

Comment:
taken straight from your code in the `Manipulator` constructor
```suggestion
        motor.restoreFactoryDefaults();
        motor.setIdleMode(CANSparkMax.IdleMode.kBrake);
        motor.setPeriodicFramePeriod(CANSparkMaxLowLevel.PeriodicFrame.kStatus0, 10);
        motor.setPeriodicFramePeriod(CANSparkMaxLowLevel.PeriodicFrame.kStatus1, 20);
```

Reply:
also no pid :)